rdr_service/dao/physical_measurements_dao.py

```diff
@@ -529,6 +530,14 @@ def from_observation(observation, full_url, qualifier_map, first_pass):
             value_coding = PhysicalMeasurementsDao.get_preferred_coding(observation.valueCodeableConcept)
             value_code_system = value_coding.system
             value_code_value = value_coding.code
+
+            value_code_description = observation.valueCodeableConcept.text
+            desc_char_count = len(value_code_description)
+            char_limit = Measurement.valueCodeDescription.type.length
+            if desc_char_count > char_limit:
+                logging.warning(f'Truncating codeable concept description of length {desc_char_count}')
+                value_code_description = value_code_description[:char_limit]
+
```

Comment:
I see that the field on the model is set to `512`,  should it be bigger since you are having to do this check? Is it okay that the value truncated?

Reply:
I set up the check as a safeguard to make sure it doesn't crash if it's too long (I don't think sqlalchemy protects us from an exception if it's too long but I haven't verified that). So I'm not expecting anything to get truncated. But if it does we'll become aware of it and be able to resolve it while the physical measurements is still able to get into the system and be used.